src/test/java/bio/terra/common/fixtures/ConnectedOperations.java

```diff
@@ -51,11 +51,7 @@
 import org.springframework.test.web.servlet.MockMvc;
 import org.springframework.test.web.servlet.MvcResult;
 
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.UUID;
+import java.util.*;
```

Comment:
```suggestion
import java.util.ArrayList;
import java.util.HashMap;
import java.util.List;
import java.util.Map;
import java.util.UUID;
```

Nit: I think it's a team norm that we write out all dependencies.